src/main/java/org/terasology/climateConditions/HypothermiaSystem.java

```diff
@@ -29,7 +29,7 @@
 
 @RegisterSystem(value = RegisterMode.AUTHORITY)
 public class HypothermiaSystem extends BaseComponentSystem {
-    private float thresholdHeight = 60f;
+    private static final float thresholdHeight = 60f;
```

Comment:
I think with with `static final` Checkstyle would complain that it is a constant now, and should be all caps. I'd leave this change out of this PR.
```suggestion
    private final float thresholdHeight = 60f;
```